src/models/Team.js

```diff
@@ -22,6 +22,10 @@ let teamSchema = new Schema({
         type: Number,
         required: true,
     },
+    profPicPath: {
+        type: String,
+        default: "",
```

Comment:
Empty string should not be used to indicate no profile picture; it should be something like `null` instead.